packages/nibijs/src/tx/signingClient.test.ts

```diff
@@ -52,6 +52,10 @@ describe("nibid tx bank send", () => {
       400000
     )
     assertIsDeliverTxSuccess(resp)
+
```

Comment:
Needed a real tx hash on the chain to make an integration test for `getTxByHash`, so I've added it here using the "bank send" tx